### DISCUSSION TITLE
Weird problem with android app from external network

### DISCUSSION THREAD

**Discussion by Nitrowolf** [advplyr/audiobookshelf#1099] [Q&A]
I'm trying to get the Android app to work with the server and having a hard time. It works just fine from outside the network in a browser and I can login and everything behaves fine. I've also tried the iOS app and it works fine as well. 

If I try to connect the android app, it immediately says "Failed to ping server." 

I'm not really sure what to do at this point. This is on a Samsung S22U.

If I try to connect from inside the network to the IP directly, it works. Since everything else BUT the android app works from outside the network, I'm lost.

**Comment by digiSal** [comment]
You need to open the port on your router to access from the outside.

**Reply by Nitrowolf** [reply]
While I didn't put it in the OP, I figured that was a given since I mentioned it was working with the iOS app and a desktop browser.

**Reply by advplyr** [reply]
There have been plenty of others that had this issue but the resolution is usually different. I've setup multiple servers never having this problem and all the popular reverse proxy solutions have been successfully used by others.
This issue brought up yesterday may be what you are experiencing: https://github.com/advplyr/audiobookshelf-app/issues/445

**Comment by Guruleenyc** [comment]
The Android app is working beautifully for me internally and externally, I have it set up with nginx and no issues to report. I wasn't aware that an IOS app was also released, which would be good for my wife 😉
